rel/ballcolor.cpp

```diff
@@ -14,22 +14,28 @@
 
 namespace ballcolor {
 
-static u8 convert_to_ballColorID(u8 color_choice){
-    if(color_choice == 0) return 3;
-    else return color_choice - 1;
+static u8 s_convert_to_ball_color_id(u8 color_choice) {
```

Comment:
Static functions should _not_ start with `s_`, unlike static vars. The reasoning is that, since static vars are mutable it's more tricky to keep track of what's accessing/modifying them, but static functions don't change, not much different than non-static